src/sflow/hsflowd/patch/dropmon/0001-sflow-enabled-drop-monitor-support-for-SONiC.patch

```diff
@@ -1,49 +1,32 @@
-From 4bdd892662c08a396066ba6a1c55eac3f8aa0a5f Mon Sep 17 00:00:00 2001
-From: Vadym Hlushko <vadymh@nvidia.com>
-Date: Tue, 25 Jan 2022 12:59:40 +0000
-Subject: [PATCH] [sflow] enabled drop monitor support for SONiC
+commit 530a0cfcefa88208b3527e0d1b6b43cf21d7dc20
+Author: marvell <marvell@cpss-build2.marvell.com>
+Date:   Mon Jul 24 16:06:57 2023 +0530
 
-Signed-off-by: Vadym Hlushko <vadymh@nvidia.com>
----
- src/Linux/Makefile                   | 2 +-
- src/Linux/hsflowd.c                  | 8 ++++++++
- src/Linux/scripts/hsflowd.conf.sonic | 2 ++
- 3 files changed, 11 insertions(+), 1 deletion(-)
+    [PATCH] [sflow] enabled drop monitor support for SONiC
+    
+    Signed-off-by: Rajkumar P R<rpennadamram@marvell.com>
 
-diff --git a/src/Linux/Makefile b/src/Linux/Makefile
-index 8128cf2..cf538e7 100644
---- a/src/Linux/Makefile
-+++ b/src/Linux/Makefile
-@@ -34,7 +34,7 @@ FEATURES_DENT= DENT PSAMPLE SYSTEMD DROPMON
- FEATURES_EOS= EAPI
- FEATURES_OS10= OS10 DBUS SYSTEMD
- FEATURES_OPX= OPX DBUS SYSTEMD
--FEATURES_SONIC= SONIC PSAMPLE DOCKER
-+FEATURES_SONIC= SONIC PSAMPLE DOCKER DROPMON
- FEATURES_XEN= XEN OVS
- FEATURES_HOST= NFLOG PCAP TCP DOCKER KVM OVS DBUS SYSTEMD
- 
 diff --git a/src/Linux/hsflowd.c b/src/Linux/hsflowd.c
-index 5d94e79..25031d1 100644
+index a29da54..4b6acc1 100644
 --- a/src/Linux/hsflowd.c
 +++ b/src/Linux/hsflowd.c
-@@ -1877,6 +1877,14 @@ extern "C" {
+@@ -1922,6 +1922,14 @@ extern "C" {
      sp->psample.ingress = YES;
-     sp->psample.egress = NO;
-     sp->psample.group = 1;
+     sp->psample.egress = YES;
+     sp->psample.group = 1; // Ingress PSAMPLE group number. Expects egress on (group+1).
 +    // drop-monitor support
 +    myLog(LOG_INFO, "drop-monitor support for SONiC");
 +    sp->dropmon.dropmon = YES;
-+    sp->dropmon.group = 1;
```

Comment:
Why sp->dropmon.group=1 removed?

Reply:
struct variable "group" is removed in version 2.0.45. This was resulting in compilation failure. Hence removed it.